@xen-orchestra/backups/_runners/VmsRemote.mjs

```diff
@@ -19,6 +19,7 @@ const DEFAULT_REMOTE_VM_SETTINGS = {
   exportRetention: 0,
   healthCheckSr: undefined,
   healthCheckVmsWithTags: [],
+  healthCheckTimeout: '10m',
```

Comment:
Shouldn't common settings be in `DEFAULT_SETTINGS` instead?

Reply:
DEFAULT_SETTINGS also applies to MetadataBackup if my understanding is correct

Reply:
Makes sense.

Maybe we should regroup VM backup settings as a lot of them are duplicated.

Reply:
it could be a good idea, less duplication but more fragmentation to find where does a setting comes from
Anyway, I feel that it's out of this PR scope

Reply:
Completely agree with you :slightly_smiling_face: